pkg/didcomm/transport/http/inbound_test.go

```diff
@@ -189,6 +189,12 @@ func TestInboundTransport(t *testing.T) {
 	})
 }
 
+func TestPanicStartWithUnsplittableServerAddr(t *testing.T) {
```

Comment:
Can you add tests to cover different hostname patterns, like ones without port(DNS), ":PORT", etc

Reply:
I added some tests. Let me know if that was what you were expecting or if you were looking for something else